Exception_demo.py

```diff
@@ -0,0 +1,47 @@
+print("Enter the num1")
+num1 = int(input())
+print("Enter the num2")
+num2 = int(input())
+try:
+    print("the sum of these two numbers is", num1+num2)
+except xception as e:
+	print(e)
+
+
+# Python program to handle simple runtime error
+#Python 3
+
+a = [1, 2, 3]
+try:
+	print ("Second element = %d" %(a[1]))
+
+	# Throws error since there are only 3 elements in array
+	print ("Fourth element = %d" %(a[3]))
+
+except:

```

Comment:
this is wrong practice of writing except clause without defining a exception it is going to handle